K2Field.K2NE.ServiceBroker/ServiceObjects/Management API/ErrorLogSO.cs

```diff
@@ -164,5 +184,43 @@ private void GetErrors()
 
 
         }
+
+        private void getErrorById()
+        {
+            string profile = "All";
+
+            base.ServiceBroker.Service.ServiceObjects[0].Properties.InitResultTable();
+            DataTable results = base.ServiceBroker.ServicePackage.ResultTable;
+
+            WorkflowManagementServer mngServer = new WorkflowManagementServer();
+            using (mngServer.CreateConnection())
+            {
+                mngServer.Open(BaseAPIConnectionString);
+
+                ErrorProfile prof = mngServer.GetErrorProfile(profile);
+                ErrorLogs errors = mngServer.GetErrorLogs(prof.ID);
+                string ErrorId = base.GetStringProperty(Constants.SOProperties.ErrorLog.ErrorId, true);
```

Comment:
There's a GetIntPrpoerty which would work better here. Avoids the convert later on.